packages/phone-number-privacy/signer/scripts/local-load-test.ts

```diff
@@ -0,0 +1,15 @@
+// tslint:disable: no-console
+
+async function start() {
+  // TODO
```

Comment:
what is this suppose to do?
Are we trying to tests something here or is something we forgot to remove?

Reply:
something we forgot to remove